test/Microsoft.OpenAPI.OData.Reader.Tests/Edm/ODataPathProviderTests.cs

```diff
@@ -48,7 +48,7 @@ public void GetPathsForGraphBetaModelReturnsAllPaths()
 
             // Assert
             Assert.NotNull(paths);
-            Assert.Equal(4887, paths.Count());
+            Assert.Equal(17313, paths.Count());
```

Comment:
The path number increases almost 5 times the original number?

Reply:
Because of all the valid containment navigation properties that are now being expanded.